src/main/scala/chisel/lib/dclib/DCOperator.scala

```diff
@@ -0,0 +1,42 @@
+package chisel.lib.dclib
+
+import chisel3._
+import chisel3.util._
+
+/**
+ * This module declares a multi-input decoupled operator.  This is an example using showing how to use
+ * DCInput and DCOutput modules to create a module with registered-output timing.
+ *
+ * @param n      The number of inputs for the operator
+ * @param width  The bit-width of inputs and outputs
+ * @param op     Function with the required operator
+ */
+class DCOperator(n: Int, width: Int, op: (UInt, UInt) => UInt) extends Module {
```

Comment:
The PR could consider calling this `DCReduce` as I believe it's supposed to be doing a reduction. Note: the `op` parameter doesn't appear to be used below.

Does it make sense to make this parametric like the other `DC*` modules?

Also, where does this PR use the `op` parameter?

Reply:
The original code used the op parameter but I had some trouble getting it working and swapped out for a hard-coded operator.  Once I got this resolved forgot to swap back.

Reply:
Resolved by 3384da4